rhods-operator/instance/base/datasciencecluster.yaml

```diff
@@ -1,7 +1,7 @@
 kind: DataScienceCluster
 apiVersion: datasciencecluster.opendatahub.io/v1
 metadata:
-  name: default
+  name: default-dsc
```

Comment:
This is a breaking change for anyone that is already using this as a reference.  I would highly advise against renaming this unless absolutely necessary.

Reply:
yeah, i was not sure if it was done with intention to keep "default" or a mistake. 
and i agree, this will cause upgrade problem, i will revert it